modules/gitops-logging-into-keycloak.adoc

```diff
@@ -0,0 +1,65 @@
+[id="gitops-logging-into-keycloak_{context}"]
+= Logging into Keycloak
+
+By logging in to the Keycloak console after configuring the Argo CD instance, you can manage identities or roles and define the permissions assigned to the various roles.
+
+.Procedure
+
+. Get the Keycloak Route URL for login:
++
+[source,terminal]
+----
+$ oc -n argocd get route keycloak
+
+NAME        HOST/PORT                                                        PATH   SERVICES   PORT    TERMINATION   WILDCARD
+keycloak    keycloak-default.apps.ci-ln-******.origin-ci-int-aws.dev.**.com         keycloak   <all>    reencrypt     None
+----
+. Get the Keycloak pod name that stores the user name and password as environment variables:
++
+[source,terminal]
+----
+$ oc -n argocd get pods
+
+NAME                      READY   STATUS           RESTARTS   AGE
+keycloak-1-2sjcl           1/1    Running            0        45m
+----
+.. Get the Keycloak user name:
++
+[source,terminal]
+----
+$ oc -n argocd exec keycloak-1-2sjcl -- "env" | grep SSO_ADMIN_USERNAME
+
+SSO_ADMIN_USERNAME=Cqid54Ih
+----
+.. Get the Keycloak password:
++
+[source,terminal]
+----
+$ oc -n argocd exec keycloak-1-2sjcl -- "env" | grep SSO_ADMIN_PASSWORD
+
+SSO_ADMIN_PASSWORD=GVXxHifH
+----
+. On the login page, click *LOG IN VIA KEYCLOAK*.
++
+[NOTE]
+====
+You only see the option *LOGIN VIA KEYCLOAK* after the Keycloak instance is ready.
+====
+. Click *Login with OpenShift*.
+. Enter the OpenShift credentials to log in.
++
+[NOTE]
+====
+Login using `kubeadmin` is not supported.
+====
```

Comment:
This note seems apt after the previous step: Click *Login with OpenShift*.
It explains why the user should not click Login using `kubeadmin`